util/reggen/validate.py

```diff
@@ -385,7 +385,10 @@ def bitmask(bfield):
         "register name should be given here. empty-string for no register " +
         "write protection"
     ],
-    'resval': ['d', "reset value of full register (default 0)"]
+    'resval': ['d', "reset value of full register (default 0)"
+    ],
+    'tags': ['s', "tags for the register, followed by the format 'tag_name:item1:item2...'"]
```

Comment:
Not only here but it would be good to add the field description and the list of values in the register tool document.

Reply:
Do you mean the `doc/rm/register_tool/index.md`? I have already updated that doc, is there any other description file that i missed? Also for the future step, we are considering adding this tag into the register table for easier review process.

Reply:
Oh... I just scrolled down. So missed that part :) Thanks

Reply:
Thanks again for double-checking for me. No worries, this PR is getting too long..